src/proc_image_processing/cpu/sonar/SonarMapper.h

```diff
@@ -2,25 +2,6 @@
  * \file	SonarMapper.h
  * \author	Jeremie St-Jules-Prevost <jeremie.st.jules.prevost@gmail.com>
  * \date	06/02/2016
- *
- * \copyright Copyright (c) 2015 S.O.N.I.A. All rights reserved.
```

Comment:
I did not check the files in the folder sonar when I did my first cleanup of the module since we determined that this was out of the project's scope, but you can modify them if you want it.